src/components/ChatInput.tsx

```diff
@@ -1,7 +1,7 @@
 import { Send, Loader2, Settings } from 'lucide-react';
 import { Button } from '@/components/ui/button';
 import { Textarea } from '@/components/ui/textarea';
-import { useState, type FC, type FormEvent, type KeyboardEvent } from 'react';
```

Comment:
Just testing (this shouldn't show in testing of the gh pr script, since its a resolved comment).